test/t8_test_cmesh_readmshfile.c

```diff
@@ -0,0 +1,164 @@
+#include <t8_eclass.h>
```

Comment:
The licence information is missing.

Look at the other files, each file in t8code needs a licence text at the beginning.